runtime/version/version_test.go

```diff
@@ -22,6 +22,23 @@ import (
 	"testing"
 )
 
+// Test that version matches Major, Minor, and Patch.
+func TestStaleVersion(t *testing.T) {
```

Comment:
I am not sure we need this test. `TestReadVersion` is a superset of this test, and it fails as well if the hardcoded version string is not updated. 

I don't have strong feelings though.

Reply:
Oh oops! I didn't realize the other tests would fail. I'll abandon the PR.